### DISCUSSION TITLE
[Feature Request] Gallery feature

### DISCUSSION THREAD

**Discussion by Nomidos** [zanllp/infinite-image-browsing#512] [Ideas]
Also 1 more thing,  IMHO, I think this plugin could benefit from a gallery that allows you to better organize the images you generate into projects. Instead of just having a massive catalog to scroll through.

**Comment by Pbillo** [comment]
This would be great! I am looking for a similar thing. 

I would love to be able to set folders that would act as "collections" that I can send images to through the browser. In my workflow i tend to have several projects running that have their own look and feel and I tend to dip in and out of them. My workaround is to set tags for these and than call them in search. But I still have to manually move them to these folders in explorer..

(edit) just to be clear, Instead of just send to saved folder have the option to send to folders I have added to the index..

**Comment by zanllp** [comment]
@Pbillo You can check out this feature, I'm not sure if it's what you're looking for, but actually, in the past, you could accomplish this operation through drag and drop.

 https://github.com/zanllp/sd-webui-infinite-image-browsing/wiki/Change-log#added-move-to-copy-to-copy-file-path-and-open-file-location-in-new-tab-to-context-menu

**Comment by zanllp** [comment]
I had seen this discussion before, but to be honest, I didn't quite understand your description, so I ignored it directly. Besides the walk mode, IIB also offers other modes to help you organize files, which can be found in the second column on the startpage.
